src/actions/articles/rating/rateArticleAction.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable func-names */
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { toastSuccess, toastFailure } from '../../../utils/toast';
+import { ArticleConstants } from '../../actionTypes';
+
+const apiUrl = 'https://ah-backend-kronos-staging.herokuapp.com/api/articles/';
+const lastPart = '/rate';
```

Comment:
I think we have url connections file that stores the url there is no need to declare the url again.